src/api/app/controllers/webui/request_controller.rb

```diff
@@ -299,6 +299,66 @@ def build_results
     @ajax_data['package'] = @action[:spkg] if @action[:spkg]
   end
 
+  def build_results_chart
+    success_data = {}
+    failed_data = {}
+    building_data = {}
+
+    # take all the build results for all the actions (only actions where it makes sense to have a build status)
+    @actions.where(type: [:submit, :maintenance_incident, :maintenance_release]).each do |action|
+      source_project_object = Project.find_by_name(action.source_project)
```

Comment:
@ncounter what I meant this morning regarding the build results for a request to a staging project. I think we have to consider it here as well. Basically whats done here https://github.com/openSUSE/open-build-service/blob/2b928d8e8374de93addeb4d2d30da986dcdcf54c/src/api/app/controllers/webui/request_controller.rb#L294
 So showing the build results of the staging project (the `bs_request` and `bs_request_action` then contains a `:staging_project_id`).

Reply:
Addressed, thank you for the pointer :+1: